csclient/csclient_test.go

```diff
@@ -27,29 +31,24 @@ import (
 
 	"gopkg.in/juju/charmstore.v4"
 	"gopkg.in/juju/charmstore.v4/csclient"
-	"gopkg.in/juju/charmstore.v4/internal/blobstore"
-	internalCharmstore "gopkg.in/juju/charmstore.v4/internal/charmstore"
-	"gopkg.in/juju/charmstore.v4/internal/router"
 	"gopkg.in/juju/charmstore.v4/internal/storetesting"
```

Comment:
Yeah, it can't be removed yet because testing charm repos are always local. I mentioned this in the PR description.